Sources/SwiftSyntax/generated/syntaxNodes/SyntaxDeclNodes.swift

```diff
@@ -89,7 +89,8 @@ public struct AccessorDeclSyntax: DeclSyntaxProtocol, SyntaxHashable {
           arena: arena, 
           leadingTrivia: leadingTrivia, 
           trailingTrivia: trailingTrivia
-        )
+        
+      )
```

Comment:
Weird, but I assume the new will fix this?

Reply:
I’m not entirely sure yet what’s going wrong here. Still need to investigate but I didn’t think it would make sense to investigate in the current `BasicFormat`.